config/install_debug.yaml

```diff
@@ -11998,3 +11998,64 @@ spec:
   selector:
     app: kyverno
     app.kubernetes.io/name: kyverno
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-apiserver
+  namespace: kyverno
+spec:
+  egress:
+  - ports:
+    - port: 9443
+    to:
+    - namespaceSelector:
+        matchLabels:
+          kubernetes.io/metadata.name: kube-system
+      podSelector:
+        matchLabels:
+          component: kube-apiserver
+          tier: control-plane
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+          kubernetes.io/metadata.name: kube-system
+      podSelector:
+        matchLabels:
+          component: kube-apiserver
+          tier: control-plane
+    ports:
+    - port: 9443
+  podSelector: {}
+  policyTypes:
+  - Ingress
+  - Egress
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  labels:
+    app: kyverno
+  name: allow-prometheus
+  namespace: kyverno
+spec:
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+          app.kubernetes.io/name: kyverno
+      podSelector:
+        matchLabels:
+          app.kubernetes.io/name: prometheus
+    ports:
+    - port: 9443
```

Comment:
Similar to comment on Helm chart, Prometheus doesn't need access to webhook port.